src/main/java/net/earthcomputer/clientcommands/command/CDebugCommand.java

```diff
@@ -0,0 +1,35 @@
+package net.earthcomputer.clientcommands.command;
+
+import com.mojang.brigadier.Command;
+import com.mojang.brigadier.CommandDispatcher;
+import net.fabricmc.fabric.api.client.command.v2.FabricClientCommandSource;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.components.DebugScreenOverlay;
+
+import static net.fabricmc.fabric.api.client.command.v2.ClientCommandManager.literal;
+
+public class CDebugCommand {
+    public static void register(CommandDispatcher<FabricClientCommandSource> dispatcher) {
+        dispatcher.register(literal("cdebug")
+                .executes((context -> execute("overlay")))
+                .then(literal("fps").executes(context -> execute("fps")))
+                .then(literal("network").executes(context -> execute("network")))
+                .then(literal("profiler").executes(context -> execute("profiler")))
```

Comment:
Could you format this the way other commands are formatted too? And maybe add the literal `overlay` as well; based on the suggestions people might not realise short-circuiting is an option.